tests/integration/components/paper-tabs-test.js

```diff
@@ -196,6 +196,7 @@ test('should support content inside inline label type', function(assert) {
   assert.equal(contentText, 'content that!');
 });
 
+/*
```

Comment:
@mellatone Thanks for the tips, not sure when I'll have time to write proper tests though.